frontend/app/views/spree/checkout/edit.html.erb

```diff
@@ -1,5 +1,16 @@
 <% @body_id = 'checkout-page' %>
 
+<%
+content_for :head do
+  agent = request.env['HTTP_USER_AGENT']
+  if agent =~ /Safari/ && !(agent =~ /Chrome/)
+%>
+  <meta name="turbolinks-visit-control" content="reload">
```

Comment:
This will be not working. Try this, it works in another project.
```suggestion
  if (!!window.performance && window.performance.navigation.type === 2) {
    window.location.reload();
  }

  window.onpageshow = function(event) {
    if (event.persisted) {
        window.location.reload()
    }
  }
```

Reply:
Why do You think it won't work? We used it in 3Form project when we encountered the same issue :)

Reply:
This meta tag was not working on safari browser in version < 13.1. If you are sure it will be working, it is ok.

Reply:
Tested Your solution and it seems to be working so I decided to use that if You say it is going to be compatible with older Safari browsers. Didn't have an older version to test it though :)